src/test/java/build/buildfarm/instance/memory/MemoryInstanceTest.java

```diff
@@ -225,9 +211,9 @@ public void watchOperationAddsWatcher() throws InterruptedException {
     watchers.put(operation.getName(), operationWatchers);
 
     Predicate<Operation> watcher = (Predicate<Operation>) mock(Predicate.class);
+    when(watcher.test(eq(operation))).thenReturn(true);
```

Comment:
only reason to mock a predicate is if you're going to verify it. Otherwise could just use a lambda.